frontend/awx/administration/notifiers/hooks/useNotifiersRowActions.tsx

```diff
@@ -56,7 +58,7 @@ export function useNotifiersRowActions(
         icon: RocketIcon,
```

Comment:
Not a part of this PR but I noticed the old UI uses the `BellIcon` instead of the `RocketIcon` here